docs/30-configuration/35-watches.md

```diff
@@ -0,0 +1,46 @@
+# Watches
+
+A watch is a configuration of a service to watch in Consul. The watch monitors the state of the service and emits events when the service becomes healthy, becomes unhealthy, or has a change in the number of instances. Note that a watch does not include a behavior; watches only emit the event so that jobs can consume that event.
```

Comment:
The first would make the second sentence partially redundant. I'll go with the second.